simpeg_drivers-assets/uijson/tdem_forward.ui.json

```diff
@@ -136,7 +137,12 @@
         "parent": "mesh",
         "value": ""
     },
-    "output_tile_files": false,
+    "save_sensitivities": {
```

Comment:
That doesn't belong here.

Reply:
will have to keep the parameter until we have separate fwd/inv drivers since we still ask for save_sensitivities in the directive factory for forward runs.  I've flattened to key/value so that it doesn't show up in the ui.